specs/FrameProcesssInfo.md

```diff
@@ -410,10 +372,12 @@ interface ICoreWebView2GetProcessInfosWithDetailsCompletedHandler : IUnknown {
 [uuid(982ae768-e2ca-11ed-b5ea-0242ac120002), object, pointer_default(unique)]
 interface ICoreWebView2ProcessInfo2 : ICoreWebView2ProcessInfo {
   /// Gets the collection of the assocated `FrameInfo`s which are running in that
-  /// renderer process.
-  /// This is only available when process `Kind` is `COREWEBVIEW2_PROCESS_KIND_RENDERER`.
-  /// `frames` is `null` for all other process kinds.
-  /// Note that `frames` can be `null` if there's no frame running in that process.
+  /// renderer process. Assocated `FrameInfo`s can be `null` if there's no frame 
+  /// running in that renderer process. 

```

Comment:
Can we make this an empty collection instead of a null collection? It is slightly easier to write code without having to check for null for that case.